.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+/.env
```

Comment:
can you add .env-sample with sample data, please as a ref.

Reply:
the .env-sample file has been added